grants_policy.md

```diff
@@ -0,0 +1,118 @@
+# Policty on requesting and/or acquiring grants/funding for/by the CWL Project
+
+## Policy
+
+It is a requirement of the CWL fiscal sponsor, Software Freedom Conservancy (SFC),
+that all applications by any entity for
+1) funding of the CWL project's work or
+2) in the name of the CWL project
+must be submitted by SFC **unless** an agreement otherwise has been approved by
+both the CWL Project Leadership Team and SFC.
+
+Submitting grants and/or getting funding to do the work of the CWL
+Project (or otherwise in the name of or on-behalf of the CWL Project) without
+approval from SFC and the CWL Leadership Team can result in cancellation of
+sustaining/support member status (without refund), censure, and/or other
+consequences as deemed appropriate by SFC and the CWL Leadership Team.
+
+Proposals for implementations (or enhancements of implementations) of the CWL
+Standards do not need approval, unless it is of the CWL reference implementation
+(`cwltool`).
+
+Proposing work related to any of the CWL GitHub organizations is a sign that
+approval **is needed**.
+- <https://github.com/common-workflow-language>
+- <https://github.com/common-workflow-lab>
+- <https://github.com/common-workflow-library>
+
+If there is doubt about what constitutes "the work of the CWL project", please
+ask the CWL Leadership Team directly.
+
+## Motivation
+
+CWL is a member project of Software Freedom Conservancy, Inc. As per the
+[membership agreement signed by all members of the initial CWL project
+leadership
+team](https://github.com/common-workflow-language/governance/blob/ddb07b99ae62006f70cd43987843626ffe08c2f0/CWL-sponsorship-agreement--signed-by-all.pdf),
+CWL is a [direct
+project](https://en.wikipedia.org/wiki/Fiscal_sponsorship#Table_comparing_the_models)
+of SFC: CWL is owned by SFC, and the project's work is implemented by
+SFC employees, volunteers, and contractors. Therefore this policy exists to document
+that requirement and how we will manage it.
+
+## Implementation
+
+This section documents how an entity (the Properser) submitting grants
+that involve Common Workflow Language standards (CWL) and the CWL
+Project should seek approval for their proposal from the CWL Leadership team and
+SFC.
+
+1.  If a proposal indicates the use of CWL but will *not* be contributing
+    substantially to the CWL Project (e.g. code, documentation, tutorials), then
+    **Proposal approval is needed**.
```

Comment:
This was discussed in 2022-04-20 leadership meeting and I thought we agree this was typo for _is **not** needed_.  

It would be unreasonable to require approval for any project proposal that use CWL! We want researchers to use CWL.

Reply:
Fixed, thanks!